experimental/packages/opentelemetry-sdk-node/src/types.ts

```diff
@@ -14,27 +14,28 @@
  * limitations under the License.
  */
 
-import { SpanAttributes, TextMapPropagator, Sampler } from '@opentelemetry/api';
 import type { ContextManager } from '@opentelemetry/api';
+import { Sampler, Attributes, TextMapPropagator } from '@opentelemetry/api';
 import { InstrumentationOption } from '@opentelemetry/instrumentation';
+import { Resource, ResourceAttributes } from '@opentelemetry/resources';
 import { MetricExporter, Processor } from '@opentelemetry/sdk-metrics-base';
-import { Resource } from '@opentelemetry/resources';
 import {
   SpanExporter,
-  SpanProcessor,
   SpanLimits,
+  SpanProcessor,
 } from '@opentelemetry/sdk-trace-base';
 
 export interface NodeSDKConfiguration {
   autoDetectResources: boolean;
   contextManager: ContextManager;
-  defaultAttributes: SpanAttributes;
+  defaultAttributes: Attributes;
```

Comment:
This one should be kept `SpanAttributes` to support the use of new SDK version with old api versions (1.0.x)

Reply:
Has this been fixed?

Reply:
Oops no